### PR TITLE
Incorrect doxygen comments fixed (no parameter description)

### PR DESCRIPTION
**Pull request by certaindividual**
For cBinder project, it was necessary to add parameter descriptions to doxygen comments, where they were omitted.

### CODE REVIEW

File: include/AT_DataParticle.h [hunk 1]
```diff
@@ -317,7 +317,7 @@ int AT_particle_name_from_particle_no( const long  n,
  * Returns particle names for given particle numbers
  * @param[in]  n              TODO
  * @param[in]  particle_name  particle names
- * @param[out] corresponding  particle numbers
+ * @param[out] particle_no    TODO
```
Comment (grzanka) [inline include/AT_DataParticle.h]: Change to:
```
 * @param[in]  particle_no  type of the particles
 * @see          AT_DataParticle.h for definition
```

File: include/AT_EnergyLoss.h [hunk 2]
```diff
@@ -177,7 +177,11 @@ void AT_lambda_landau_from_energy_loss_multi( const long n,
 		double lambda_landau[]);
 
 /**
- *
+ * @param[in]  	   energy_loss_keV          TODO
+ * @param[in]  	   E_MeV_u      		    TODO
+ * @param[in]  	   particle_no      		TODO
```
Comment (grzanka) [inline include/AT_EnergyLoss.h]: Change to:
```
 * @param[in]  particle_no  type of the particles
 * @see          AT_DataParticle.h for definition
```

File: include/AT_EnergyLoss.h [hunk 3]
```diff
@@ -357,7 +361,11 @@ void AT_lambda_vavilov_from_energy_loss_multi( const long n,
 		double lambda_vavilov[]);
 
 /**
- *
+ * @param[in]  	   energy_loss_keV          TODO
+ * @param[in]  	   E_MeV_u      		    TODO
+ * @param[in]  	   particle_no      		TODO
```
Comment (grzanka) [inline include/AT_EnergyLoss.h]: Change to:
```
 * @param[in]  particle_no  type of the particles
 * @see          AT_DataParticle.h for definition
```

File: include/AT_RDD_Tabulated.h [hunk 4]
```diff
@@ -105,6 +105,8 @@ double	 AT_r_max_RadicalDiffusion_m( const double E_MeV_u);
  * Returns minimal dose for the radical diffusion model
  *
  * @param[in]   E_MeV_u		energy in MeV/u
+ * @param[in]   particle_no		TODO
```
Comment (grzanka) [inline include/AT_RDD_Tabulated.h]: Change to:
```
 * @param[in]  particle_no  type of the particles
 * @see          AT_DataParticle.h for definition
```

File: include/AT_RDD_Tabulated.h [hunk 5]
```diff
@@ -115,6 +117,8 @@ double	 AT_d_min_RadicalDiffusion_Gy( const double E_MeV_u,
  * Returns maximum dose for the radical diffusion model
  *
  * @param[in]   E_MeV_u		energy in MeV/u
+ * @param[in]   particle_no		TODO
```
Comment (grzanka) [inline include/AT_RDD_Tabulated.h]: Change to:
```
 * @param[in]  particle_no  type of the particles
 * @see          AT_DataParticle.h for definition
```

File: include/AT_RDD_Tabulated.h [hunk 6]
```diff
@@ -115,6 +117,8 @@ double	 AT_d_min_RadicalDiffusion_Gy( const double E_MeV_u,
...
+ * @param[in]   material_no		TODO
```
Comment (grzanka) [inline include/AT_RDD_Tabulated.h]: Change to:
```
 * @param[in]  material_no  material index
 * @see          AT_DataMaterial.h for definition
```

File: include/AT_RDD_Tabulated.h [hunk 7]
```diff
@@ -105,6 +105,8 @@ double	 AT_r_max_RadicalDiffusion_m( const double E_MeV_u);
  * Returns minimal dose for the radical diffusion model
  *
  * @param[in]   E_MeV_u		energy in MeV/u
+ * @param[in]   particle_no		TODO
+ * @param[in]   material_no		TODO
```
Comment (grzanka) [inline include/AT_RDD_Tabulated.h]: Change to:
```
 * @param[in]  material_no  material index
 * @see          AT_DataMaterial.h for definition
```

File: include/AT_EnergyLoss.h [hunk 8]
```diff
@@ -357,7 +361,11 @@ void AT_lambda_vavilov_from_energy_loss_multi( const long n,
...
+ * @param[in]  	   material_no      		TODO
```
Comment (grzanka) [inline include/AT_EnergyLoss.h]: Change to:
```
 * @param[in]  material_no  material index
 * @see          AT_DataMaterial.h for definition
```

File: include/AT_EnergyLoss.h [hunk 9]
```diff
@@ -177,7 +177,11 @@ void AT_lambda_landau_from_energy_loss_multi( const long n,
 		double lambda_landau[]);
 
 /**
- *
+ * @param[in]  	   energy_loss_keV          TODO
+ * @param[in]  	   E_MeV_u      		    TODO
+ * @param[in]  	   particle_no      		TODO
+ * @param[in]  	   material_no      		TODO
```
Comment (grzanka) [inline include/AT_EnergyLoss.h]: Change to:
```
 * @param[in]  material_no  material index
 * @see          AT_DataMaterial.h for definition
```

File: include/AT_EnergyLoss.h [hunk 10]
```diff
@@ -177,7 +177,11 @@ void AT_lambda_landau_from_energy_loss_multi( const long n,
 		double lambda_landau[]);
 
 /**
- *
+ * @param[in]  	   energy_loss_keV          TODO
+ * @param[in]  	   E_MeV_u      		    TODO
```
Comment (grzanka) [inline include/AT_EnergyLoss.h]: Change to
```
* @param[in]  E_MeV_u      energy of particle per nucleon [MeV/u]
```

File: include/AT_EnergyLoss.h [hunk 11]
```diff
@@ -357,7 +361,11 @@ void AT_lambda_vavilov_from_energy_loss_multi( const long n,
 		double lambda_vavilov[]);
 
 /**
- *
+ * @param[in]  	   energy_loss_keV          TODO
+ * @param[in]  	   E_MeV_u      		    TODO
```
Comment (grzanka) [inline include/AT_EnergyLoss.h]: Change to
```
* @param[in]  E_MeV_u      energy of particle per nucleon [MeV/u]
```

File: include/AT_DataParticle.h (hunk 1 above)
Comment (certaindividual) [reply include/AT_DataParticle.h]: Should this @see annotation be inserted into AT_DataParticle.h itself?

Comment (grzanka) [reply include/AT_DataParticle.h]: good point, not necessary here

**Pull request merged**